setpoint_max_power : 100
setpoint_min_power : -100
setpoint_current_power: 0
